src/main/java/seedu/address/model/patient/Patient.java

```diff
@@ -20,22 +20,32 @@ public class Patient {
     private final Name name;
     private final Phone phone;
     private final Email email;
+    private final Gender gender;
+    private final IcNumber icNumber;
+    private final Birthday birthday;
 
     // Data fields
     private final Address address;
     private final Set<Tag> tags = new HashSet<>();
-
+    private final Department department;
```

Comment:
For this field, should it be `AssignedDepartment.java` instead? Since Department is just an enum.